packages/utilities/psammead-calendars/src/calendars/jalaali.js

```diff
@@ -0,0 +1,56 @@
+const jalaaliJs = require('jalaali-js');
+
+const jalaaliMonths = {
+  fa: [
+    'فروردین',
+    'اردیبهشت',
+    'خرداد',
+    'تیر',
+    'مرداد',
+    'شهریور',
+    'مهر',
+    'آبان',
+    'آذر',
+    'دی',
+    'بهمن',
+    'اسفند',
+  ],
+  ps: [
+    'وری',
+    'غویی',
+    'غبرګولی',
+    'چنګاښ',
+    'زمری',
+    'وږی',
+    'تله',
+    'لړم',
+    'لیندۍ',
+    'مرغومی',
+    'سلواغه',
+    'کب',
+  ],
+};
+
+const jalaali = {
+  formatDate: gregorianMoment => {
+    if (typeof gregorianMoment.locale !== 'function') {
```

Comment:
If `gregorianMoment` isn't defined, this line might throw a type error.

Reply:
Yeah you're right it doesn't throw an error, I'll amend